package.json

```diff
@@ -50,5 +50,393 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
```

Comment:
How did you end up with so many dependencies here?

Reply:
oh that's where that went! used ' npm init -y' before I entered the right folder. whoops.